.gitignore

```diff
@@ -1,21 +1,22 @@
-.sconsign.dblite
-config.log
-.sconf_temp
-*.pyc
-.vscode
-
-# Built by sconsUtils
-version.py
-
-# OpenSplice logs
-ospl-*.log
-
-# Pytest
-tests/.tests
-pytest_session.txt
-.cache/
-.pytest_cache
-.coverage
-
-# vim
-*.swp
+.sconsign.dblite

```

Comment:
... I don't... but I updated this with sourcetree rather than manually... i wonder if it changed the permissions, or line endings... I can try to change it back.